crates/bevy_ecs/src/system/commands/parallel_scope.rs

```diff
@@ -0,0 +1,93 @@
+use std::cell::Cell;
+
+use thread_local::ThreadLocal;
+
+use crate::{
+    entity::Entities,
+    prelude::World,
+    system::{SystemParam, SystemParamFetch, SystemParamState},
+};
+
+use super::{CommandQueue, Commands};
+
+#[derive(Default)]
+pub struct ParallelCommandsState {
```

Comment:
```suggestion
/// The internal state of a [`ParallelCommands`] object
pub struct ParallelCommandsState {
```

Reply:
Unfortunately, it appears that this must be pub, because it's the associated type for a public trait.